tools/mo/openvino/tools/mo/load/tf/loader.py

```diff
@@ -7,6 +7,10 @@
 os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
 try:
     import tensorflow.compat.v1 as tf_v1
+
+    import numpy
+    print("+++++++++++++++++++++++++++++++++++++++++++++NUMPY version = ", numpy.__version__)
```

Comment:
That looks as a workaround. Can you do it maybe in the common place where MO prints parameters?

Reply:
Looks like this is done for debug purpose according to commit message. @rkazants  should we put the PR to draft until it is ready then?

Reply:
@andrei-kochin, I put do_not_merge label